.github/workflows/Tests.yml

```diff
@@ -0,0 +1,39 @@
+name: "Run Tests"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref_name != github.event.repository.default_branch || github.ref != 'refs/tags/v*' }}
+
+jobs:
+  tests:
+    name: "Tests"
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1'
+        arch:
+          - x64
+          - x86
+        group:
+          - Interface
+          - Integrators
+          - Regression
+        include:
+          - coverage: false
+          - version: '1'
+            arch: x64
+            coverage: true
+    uses: "SciML/.github/.github/workflows/tests.yml@v1"
```

Comment:
It's not missing a - here?

Reply:
No, this syntax is correct, as evidenced by the workflows running as expected.